docs/04-how-we-work/tools/google-calendar.md

```diff
@@ -13,8 +13,8 @@
 ## Best Practices
 
 *   Set your location, time zone and working hours (& keep them current) using calendar settings: ![Calendar Settings](../../images/CivicActions_Calendar_Settings.png "Calendar settings")
-*   Always send invitations to <mailto:first.last@civicactions.com>
-*   Make sure your main <mailto:first.last@civicactions.com> calendar is at sharing at least free/busy with everyone in CivicActions
+*   Always send invitations to [first.last@civicactions.com](mailto:first.last@civicactions.com)
```

Comment:
Maybe better if this is `first.last@civicactions.com` rather than a real email address that will bounce.

The rest of the change in this pull request look good.